releasenotes/notes/profile-upload-2.4-8277b229fda6ff53.yaml

```diff
@@ -0,0 +1,3 @@
+other:
+  - |
+    Profiler: upload profiles using new backend request format
```

Comment:
Should we include any new agent version requirements here?

Reply:
Nom to the best of my understanding